### ISSUE TITLE
Where do i find the Documentation?

### ISSUE THREAD

**Issue by Igloczek** [victorjonsson/jQuery-Form-Validator#751]
@victorjonsson I know that you are not working on this plugin anymore, but is there a chance to put http://www.formvalidator.net/ back online or maybe host it as a static files on Github Pages?

**Comment by klsweat** [comment]
I agree. I still use this plugin, it is still functional, but w/o the documentation you can't remember all the features. Is it possible to get the static pages of the documentation?

**Comment by victorjonsson** [comment]
Well... one could always extract the content from wayback machine and put the content somewhere else.

https://web.archive.org/web/20200201115759/http://www.formvalidator.net/

**Comment by Igloczek** [comment]
Mirror created - https://igloczek.github.io/formvalidator-net-mirror/

**Comment by klsweat** [comment]
I cannot tell you how much I appreciate this. I didn't even know you that web.archive was a thing where you can pull up old sites. Thank you @victorjonsson  for teaching me something new and thank you @Igloczek for sharing your documentation
